test/test_scheduler.py

```diff
@@ -4,27 +4,29 @@
 import datetime
 import time
 import calendar
-from oncall.bin import scheduler
+import oncall.scheduler.default
 from pytz import utc, timezone
 
 MIN = 60
 HOUR = 60 * MIN
 DAY = 24 * HOUR
 WEEK = 7 * DAY
 
+MOCK_SCHEDULE = {'team_id': 1, 'role_id': 2, 'roster_id': 3}
 
 def test_find_new_user_as_least_active_user(mocker):
-    mocker.patch('oncall.bin.scheduler.find_new_user_in_roster').return_value = set([123])
-    mocker.patch('oncall.bin.scheduler.get_roster_user_ids').return_value = set([135, 123])
-    mocker.patch('oncall.bin.scheduler.get_busy_user_by_event_range')
-    mocker.patch('oncall.bin.scheduler.find_least_active_user_id_by_team')
+    scheduler = oncall.scheduler.default.Scheduler()
+    mocker.patch('oncall.scheduler.default.Scheduler.find_new_user_in_roster').return_value = set([123])
```

Comment:
to avoid the extra syntax `set([])` you can use the shorthand `{}` to make a set out of a list

eg `{123}` and `{135, 125}`